cirq-core/cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -307,7 +297,7 @@ def compile_circuit(
     # as well, we allow this to be passed in. This compiler is not allowed to
     # change the order of the qubits.
     return CompilationResult(
-        circuit=compiler(swap_networks[0].circuit), mapping=mapping, parity_map=parity_map
```

Comment:
Why was `swap_networks[0].circuit` being pass in in the first place instead of `routed_circuit`?

Reply:
Because the original optimizer mutated the circuit. `routed_circuit` was a variable storing a reference to `swap_networks[0].circuit` and since the underlying circuit was being mutated; it didn't matter whether you pass `swap_networks[0].circuit` or `routed_circuit`.

Now, `routed_circuit` stores a modified copy of `swap_networks[0].circuit` and therefore we need to pass `routed_circuit` instead of `swap_networks[0].circuit`.